resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -375,7 +375,7 @@ credentials.mfa.display=''{0}'' requires MFA to connect
 credentials.mfa.display.short=MFA code required
 credentials.mfa.message=Enter MFA code for device: {0}
 credentials.mfa.title=MFA Required to Connect to AWS Using {0}
-credentials.profile.circular_profiles=Profile ''{0}'' is invalid due to a circular profile dependency was found between {1}
+credentials.profile.circular_profiles=Profile ''{0}'' is invalid due to a circular profile dependency found between {1}
```

Comment:
There are a couple of tests in `software.aws.toolkits.jetbrains.core.credentials.ProfileReaderTest` that needs to be updated to reference the message instead of hard coding the assert

Reply:
I modified `software.aws.toolkits.jetbrains.core.credentials.ProfileReaderTest` to use the message() strings from MessagesBundle.properties .  
2 tests used the `credentials.profile.circular_profiles` string that I changed in the previous commit.  
But I also changed the other tests in the ProfileReaderTest.kt file that had hard-coded strings, so they are now all using the message() function.

The tests passed when I ran them locally on my mac using `./gradlew check`

The automated test 'Unit Test / Mac (pull_request)' finished successfully.  
There are 3 automated tests that failed for Linux and Windows ... I could be wrong about this, but I don't think they are related to my changes.

Reply:
One was a network error downloading dependencies, another was file locking issue with Gradle + Rider